package.json

```diff
@@ -5,64 +5,59 @@
     "homepage": "/tiltaksgjennomforing",
     "type": "module",
     "scripts": {
-        "build": "tsc && vite build",
+        "build": "vite build",
+        "prettier": "prettier -c .",
+        "prettier:fix": "prettier -w .",
         "release": "node sentry.js",
         "start": "vite",
+        "svgo": "sh ./svgo.sh",
         "test": "vitest --environment=jsdom",
         "up": "docker-compose up -d --remove-orphans --build"
     },
     "dependencies": {
-        "@navikt/arbeidsgiver-notifikasjon-widget": "^6.3.9",
-        "@navikt/bedriftsmeny": "^6.11.0",
-        "@navikt/ds-css": "^5.2.1",
+        "@navikt/arbeidsgiver-notifikasjon-widget": "^6.4.2",
```

Comment:
ny versjon 6.5.0 av `@navikt/arbeidsgiver-notifikasjon-widget` er tilgjengelig med en ny notifikasjonstype.

Reply:
Samme for denne, vi er på `6.5.0` - sjekk `package-lock`